app/services/datastore/application_rehydration.rb

```diff
@@ -73,11 +73,11 @@ def income
     end
 
     def dependants
-      parent.means_details&.dependants&.map { |struct| Dependant.new(**struct) } || []
+      parent.means_details&.income_details&.dependants&.map { |struct| Dependant.new(**struct) } || []
     end
 
     def client_has_dependants
-      parent.means_details&.dependants&.any? ? YesNoAnswer::YES : YesNoAnswer::NO
+      parent.means_details&.income_details&.dependants&.any? ? YesNoAnswer::YES : YesNoAnswer::NO
```

Comment:
I'm not sure I can 100% guarantee this, but if there is a means_details there should always be an income_details, so we wouldn't need the second null safety check